components/Ripple/tests/unit/MDCRippleTouchControllerTests.m

```diff
@@ -70,4 +117,72 @@ - (void)testAddRippleToView {
                 NSStringFromCGRect(parentView.bounds));
 }
 
+- (void)testInsertRippleViewDelegateMethod {
+  // Given
+  UIView *parentView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 100, 100)];
+  MDCRippleTouchController *touchController = [[MDCRippleTouchController alloc] init];
+  FakeMDCRippleTouchControllerDelegate *delegate =
+      [[FakeMDCRippleTouchControllerDelegate alloc] init];
+  touchController.delegate = delegate;
+  delegate.rippleTouchController = touchController;
+
+  // When
+  [parentView addSubview:[[UIView alloc] init]];
+  [touchController addRippleToView:parentView];
+
+  // Then
+  XCTAssertEqualObjects(parentView.subviews[0], touchController.rippleView);
+  XCTAssertTrue(delegate.insertRippleViewCalled);
+}
+
+- (void)testDidProcessRippleViewDelegateMethod {
+  // Given
+  UIView *parentView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 100, 100)];
+  MDCRippleTouchController *touchController = [[MDCRippleTouchController alloc] init];
+  FakeMDCRippleTouchControllerDelegate *delegate =
+      [[FakeMDCRippleTouchControllerDelegate alloc] init];
+  touchController.delegate = delegate;
+  delegate.rippleTouchController = touchController;
+  delegate.shouldProcessRippleTouchesAtTouchLocation = YES;
+
+  // When
+  [parentView addSubview:[[UIView alloc] init]];
+  [touchController addRippleToView:parentView];
+  FakeGestureRecognizer *gestureRecognizer = [[FakeGestureRecognizer alloc] initWithTarget:nil
+                                                                                    action:nil];
+  gestureRecognizer.state = UIGestureRecognizerStateBegan;
+  [touchController handleRippleGesture:gestureRecognizer];
+
+  // Then
+  XCTAssertTrue(delegate.didProcessRippleViewCalled);
+}
+
+- (void)testShouldProcessRippleViewDelegateMethod {
```

Comment:
Please adjust this name to indicate what the test is testing, similar to the suggestion above.